bedrock/firefox/templates/firefox/firstrun/new-firstrun.html

```diff
@@ -12,6 +12,8 @@
   {% javascript 'firefox_new_firstrun' %}
 {% endblock %}
 
+{% block global_nav %}{% endblock %}
```

Comment:
I noticed this template does not inherit from the firefox app, so it was getting the nav incorrectly.